package.json

```diff
@@ -4,7 +4,10 @@
   "description": "A CORS proxy for everyone",
   "main": "index.js",
   "scripts": {
-    "test": "make test"
+    "test": "npm run lint && npm run unit",
```

Comment:
if you merge the other one, i will rebase this one onto v2, which hopefully should make github happy